web/js/scripts.js

```diff
@@ -27,12 +27,16 @@ var grouphub = (function ($) {
     var searchUsersOrGroups = function () {
         var $this = $(this),
             $searchContainer = $this.closest('.search_container'),
-            $searchResults = $searchContainer.next('ul');
-
-        var url = $searchResults.data('url');
-        var type = $searchContainer.find("select[name=search-type]").val();
-        if (type === 'group') {
-            url = url.replace('users', 'groups');
+            $searchResults = $searchContainer.next('ul'),
+            $searchTab = $searchResults.parent(),
+            url = $searchResults.data('url'),
+            searchType;
+
+        if ($searchTab.attr('id') === 'add_members_tab') {
+            var searchType = $searchContainer.find("select[name=search-type]").val();
```

Comment:
Is `var` really needed here? You have already declared it in top of this function.